console/console-init/ui/src/modules/project/containers/ProjectListContainer/ProjectListContainer.tsx

```diff
@@ -29,7 +29,7 @@ import {
   RETURN_ALL_PROJECTS,
   DOWNLOAD_CERTIFICATE
 } from "graphql-module";
-import { IAddressSpace } from "modules/messaging-project";
+// import { IAddressSpace } from "modules/address-space";
```

Comment:
please remove this comment as address-space module will not exist

Reply:
Agreed.